search-endpoint.md

```diff
@@ -202,6 +209,17 @@ The returned JSON response will be like:
       },
       {
         "name" : "subject",
+        "facetType": "text",
+        "hasMore": true,
+        "facetLimit": 5,
+        "_links": {
+          "next": "/api/discover/facets/subject?page=1&size=5",
```

Comment:
I know that this is slightly different than how we have documented other endpoint (as we typically don't include in the example the linked resource) but I guess it is useful to keep the facet inline here
You need to add all the other parameters to the next and self link (query, scope, filters) to make the example consistent
I know that the current REST implementation is not able to support these additional parameters, I still need to look to your implementation PR to see if you have solved that but in any case the REST contract should tell that these parameters need to be here

Reply:
That's correct. Our implement does copy the query parameters and I'll also add that to the contract.